### DISCUSSION TITLE
Any way to black out screen when remoting in?

### DISCUSSION THREAD

**Discussion by joeystombaugh** [amidaware/tacticalrmm#1287] [Q&A] [answered]
When using the Take Control feature, is there any way to black out the monitor so that if I am remoting into a desktop I can still do what I need to without anyone in the area seeing what I am doing?

**Comment by dinger1986** [comment]
It's a feature request atm 416

**Comment by wh1te909** [comment, marked as answer]
yea this has actually been requested multiple times. but you are in the wrong repo you need to ask in https://github.com/Ylianst/MeshCentral if you search through there issues you'll find multiple tickets for this request for example https://github.com/Ylianst/MeshCentral/issues/416 https://github.com/Ylianst/MeshCentral/issues/2550 https://github.com/Ylianst/MeshCentral/issues/54
